src/smtml/bitwuzla_mappings.default.ml

```diff
@@ -407,8 +407,35 @@ module Fresh_bitwuzla (B : Bitwuzla_cxx.S) : M = struct
     let of_ieee_bv eb sb bv = mk_term1_indexed2 Kind.Fp_to_fp_from_bv bv eb sb
 
     (* TODO *)
-    let to_ieee_bv _ =
-      Fmt.failwith "Bitwuzla_mappings: to_ieee_bv not implemented"
+    let to_ieee_bv x =
+      let (sign, exp, mant) = Term.value Term.IEEE_754 x in
+      let bits_str = String.concat [sign; exp; mant] in
+      let bv = Z.of_string_base 2 bits_str in
+      let total_bits = String.length bits_str in
+      mk_bv_value_int (mk_bv_sort total_bits) (Float.to_int bv)
```

Comment:
The CI is saying that `Float.to_int bv` is incorrect because `bv` has type `Z.t` and not `float`. Why not using `Z.to_int` for instance? (If you need 64 bits you can use `Z.to_int64` but then maybe you need a `mk_bv_value_int64` or something?)